extensions/amp-accordion/1.0/accordion.type.js

```diff
@@ -56,23 +57,31 @@ AccordionDef.AccordionHeaderProps;
 /**
  * @typedef {{
  *   as: (string|PreactDef.FunctionalComponent|undefined),
+ *   role: (string|undefined)
  *   className: (string|undefined),
+ *   id: (string|undefined),
  *   children: (?PreactDef.Renderable|undefined),
  * }}
  */
 AccordionDef.AccordionContentProps;
 
 /**
  * @typedef {{
- *   role: (string|undefined),
+ *   id: (string),
+ *   role: (string),
  *   onClick: (function()|undefined),
+ *   aria-controls: (string),
```

Comment:
These two are not public API though, right? They are only set privately, so I think we don't need them here. We do not expect a developer to do `<AccordionHeader aria-expanded={...}/>`.

Reply:
Per real time conversation, refactored names and added comments for non-public API here: https://github.com/ampproject/amphtml/pull/31136#issuecomment-728285011